sdcm/utils/loader_utils.py

```diff
@@ -87,29 +74,30 @@ def _run_all_stress_cmds(self, stress_queue, params):
             stress_cmds *= stress_multiplier
 
         for stress_cmd in stress_cmds:
-            params.update({'stress_cmd': stress_cmd})
-            self._parse_stress_cmd(stress_cmd, params)
+            stress_params = dict(params)
+            stress_params.update({'stress_cmd': stress_cmd})
+
+            # Due to an issue with scylla & cassandra-stress - we need to create the counter table manually
+            if 'counter_' in stress_cmd:
+                self._create_counter_table()
+
+            if 'compression' in stress_cmd:
+                if 'keyspace_name' not in stress_params:
+                    compression_prefix = re.search('compression=(.*)Compressor', stress_cmd).group(1)
+                    keyspace_name = "keyspace_{}".format(compression_prefix.lower())
+                    stress_params.update({'keyspace_name': keyspace_name})
 
             # Run all stress commands
             self.log.debug('stress cmd: {}'.format(stress_cmd))
             if stress_cmd.startswith('scylla-bench'):
                 stress_queue.append(self.run_stress_thread(stress_cmd=stress_cmd,
                                                            stats_aggregate_cmds=False,
                                                            round_robin=self.params.get('round_robin')))
-            elif stress_cmd.startswith('cassandra-harry'):
-                stress_queue.append(self.run_stress_thread(**params))
             else:
-                stress_queue.append(self.run_stress_thread(**params))
+                stress_queue.append(self.run_stress_thread(**stress_params))
```

Comment:
Where is stress_params coming from ?

Reply:
line 77.
I'm copying params (btw. name is misleading, as we use params for sct config options, while here params refer to stress parameters) so in case of modification (line 88) does not affect all the next stress commands in list